samples/system-test/detect.test.js

```diff
@@ -70,10 +70,10 @@ describe('basic detection', () => {
   });
 
   it('should detect sentiment with intent', async () => {
-    const {stdout} = await execa.shell(
+    const stdout = cp.execSync(
       `${cmd_sentiment} ${projectId} 'SESSION_ID' '${testQuery}' 'en-US'`,
       {cwd}
     );
     assert.include(stdout, 'Detected sentiment');
   });
-});
+});
```

Comment:
Let's add these newlines at the end of the files back to keep the angry red circle away :)

Reply:
Got it, thank you so much! I will remember that. I applied the changes, please let me know if anything else needs to be touched up and I will square it away :-)